mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
```

Comment:
Error handling is not consistent. In this line, if `req.params.orderId` is not a valid ObjectId, `findByIdAndDelete` will throw a CastError, but it's not caught. Consider using `mongoose.isValidObjectId` to validate the ObjectId before calling `findByIdAndDelete`.